src/update/balrog.cpp

```diff
@@ -142,23 +142,23 @@ bool Balrog::fetchSignature(NetworkRequest* initialRequest,
   }
 
   if (x5u.isEmpty() || signatureBlob.isEmpty()) {
-    logger.log() << "No signatureBlob or x5u found";
+    logger.error() << "No signatureBlob or x5u found";
     return false;
   }
 
-  logger.log() << "Fetching x5u URL:" << x5u;
+  logger.debug() << "Fetching x5u URL:" << x5u;
 
   NetworkRequest* x5uRequest = NetworkRequest::createForGetUrl(this, x5u, 200);
 
   connect(x5uRequest, &NetworkRequest::requestFailed,
           [this](QNetworkReply::NetworkError error, const QByteArray&) {
-            logger.log() << "Request failed" << error;
+            logger.error() << "Request failed" << error;
```

Comment:
I've been noticing that we get a lot of 404 responses when running the latest version, presumeably because there is nothing to upgrade to. Perhaps we should consider lowering the severity of this event (or carving out some exception for a 404).